### PR TITLE
travis: Simplify build

### PR DESCRIPTION
**Pull request by kevinji**
Hopefully this speeds up the build a bit.

### CODE REVIEW

**Comment by kevinji** [comment]
Ooh, it finally passes!

File: ci/server.sh [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+#!/bin/bash
+set -ev
 
-# start the server and capture the pid
-java -jar target/ground-0.1-SNAPSHOT.jar server conf/config.yml &
+# Build the server.
+sbt clean package
+
+# Start the server and capture the pid.
+java -jar target/scala-2.11/ground_2.11-0.1-SNAPSHOT.jar server conf/application.yml &
```
Comment (kevinji) [inline ci/server.sh]: @vsreekanti There's still something wrong with this command but I'm not too familiar with how to start the server under Play Framework; any ideas?

**Comment by kevinji** [comment]
Seems like the [Neo4j install is currently broken](https://stackoverflow.com/q/43745678/558592), so let's hold this for a bit. Tracking: neo4j/neo4j#9291.

**Comment by kevinji** [comment]
This should be good to go.

**Comment by vsreekanti** [comment]
Cherry picked some of these changes temporarily. Once we re-implement Cassandra and Neo4j, will port the rest of the changes.

**Pull request closed without merging**